src/services/resource.js

```diff
@@ -0,0 +1,100 @@
+const httpStatusCode = require('@generics/http-status')
+const resourceQueries = require('@database/queries/resources')
+const responses = require('@helpers/responses')
+const common = require('@constants/common')
+const userRequests = require('@requests/user')
+const _ = require('lodash')
+
+module.exports = class resourceHelper {
+	/**
+	 * List resources of each users
+	 * @method
+	 * @name list
+	 * @returns {JSON} - List of resources
+	 */
+
+	static async list(user_id, queryParams, pageNo, limit) {
+		try {
+			let result = {
+				data: [],
+				count: 0,
+			}
+			let sort = {
+				sort_by: 'created_at',
+				order: 'DESC',
+			}
+			let filter = {}
+			if ('filter' in queryParams && queryParams.filter.toLowerCase() === common.FILTER_ALL.toLowerCase()) {
+				filter = {
+					user_id,
+				}
+			} else {
+				if ('type' in queryParams && queryParams.type.length > 0) {
```

Comment:
@adithyadinesh0412 at a time they can select only one type or all types, hence the length check also we can remove